packages/templates/wasm/golang/polywrap.build.yaml

```diff
@@ -0,0 +1,4 @@
+format: 0.2.0
```

Comment:
We don't need to be adding these build manifests, they're not doing anything meaningful and can be omitted.